util/plugins/eu.esdihumboldt.util.test/src/eu/esdihumboldt/util/geometry/WindingOrderTest.java

```diff
@@ -77,14 +98,56 @@ public static void setUp() {
 		clockWise4 = factory.createGeometryCollection(
 				new Geometry[] { clockWise2, clockWise2WOHoles, clockWise3, r2 });
 
+		if (crs == null) {
+			try {
+				crs = CRS_CACHE.get(code);
+			} catch (Exception e) {
+				throw new IllegalStateException("Invalid CRS code", e);
+			}
+		}
+	}
+
+	/**
+	 * Test of Flip state of CRS
+	 */
+	@Test
+	public void testCRSFlipped() {
+		assertTrue(WindingOrder.isCRSFlip(crs));
```

Comment:
You should also add at least a test with a CRS where the winding order does not have to be reversed.
Otherwise this could also be a function returning `true` all the time :wink: